README.md

```diff
@@ -716,23 +708,20 @@ with the following code:
       <p class="alert alert-danger" role="alert"><%= live_flash(@flash, :error) %></p>
       <%= @inner_content %>
     </main>
-    <%= csrf_meta_tag() %>
-    <script type="text/javascript" src="<%= Routes.static_path(@socket, "/js/app.js") %>"></script>
   </body>
 </html>
+
 ```
 
-> 🏁  For the full side-by-side view of the changes made in this step,
-see:
-[commit/031e034](https://github.com/dwyl/phoenix-liveview-counter-tutorial/pull/6/commits/031e0342041483a4a1c91c65abfff0a0d08db005)
+Essentially in Phoenix LiveView there are 2 layout templates you care about: the root layout and the LiveView layout. The root layout is what gets rendered on the initial request of the application. The LiveView layout is what gets rendered as a part of the LiveView life-cycle. You can read more about [`Live Layouts`](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#module-live-layouts) in the Hex documentation.
```

Comment:
@ivymarkwell This is a good explanation. 👍 
We try to follow https://sembr.org/ in our Markdown ⬇️ 
because it makes text easier to review/edit. 📝

Reply:
✅